### ISSUE TITLE
Issue #35 reopen

### ISSUE THREAD

**Issue by jamiehowarth0** [leekelleher/umbraco-nested-content#86]
Hi guys,

Can confirm that #35 is a bug on IE10 & 11, due to the nestedcontent.services.js file being empty.
For some reason IE10+11 think that the loaded dependency is HTML (text template, probably), hence the warning about an extra `<body>` tag in F12 dev tools.

To reproduce:
Install Nested Content 0.2.0 in any Umbraco 7.2+ site.
Open backoffice in IE10 or 11.
Backoffice will load the HTML, but refuse to render the login screen.
Open package.manifest for NC, remove nestedcontent.services.js, restart app pool.
Umbraco backoffice will now load without issue.

Basically Angular thinks that the empty JS file is a text template, but IE10/11 will see an empty HTML document thus automatically inject missing tags (as it should per W3C specs), but then that breaks the entire DOM render for the single page app.

Fix: either put some dummy JS in nestedcontent.services.js or remove the file entirely from package.manifest.

**Comment by mattbrailsford** [comment]
Interesting. Nice debugging. I guess if it's empty then we might as well
just get rid.

**Comment by jamiehowarth0** [comment]
Or just put a dummy service in there for now? Cause it's noted that it's there for future use, so a dummy service wouldn't harm anything.

**Comment by mattbrailsford** [comment]
Better to just get rid. Or at the very least remove from package manifest.
Can always put back if needed. No point loading dummy code with no
function.

**Comment by leekelleher** [comment]
Closing, this was fixed in 492a88b7ff37cf33a37b7ac0a390e18608f52db2 - was released in v0.3.0
